diagnostic reports/README.md

```diff
@@ -0,0 +1,374 @@
+| Participant / agentRef / id [@root]           | The mapped [Practitioner](../practioners/README.md) or [Organisation](../organisations/README.md) referenced by `DiagnosticReport.performer[0].actor` |
+
+<details>
+    <summary>Example XML</summary>
+
+```XML
+
+<CompoundStatement classCode="CLUSTER" moodCode="EVN">
+ <id root="74F2D322-9032-4ED2-999D-0AACA87B3BC8"/>
+ <id extension="1013/HA2101105W/200203301621" root="2.16.840.1.113883.2.1.4.5.5"/>
+ <code code="16488004" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" displayName="laboratory reporting">
+  <originalText>Filed Report</originalText>
+ </code>
+ <statusCode code="COMPLETE"/>
+ <effectiveTime>
+  <center nullFlavor="NI"/>
+ </effectiveTime>
+ <availabilityTime value="20020330162100"/>
+ <component contextConductionInd="true" typeCode="COMP">
+  <NarrativeStatement classCode="OBS" moodCode="EVN">
+   <id root="E8F624FF-9364-42BB-9E01-8D3512BBE732"/>
+   <text mediaType="text/x-h7uk-pmip">CommentType:LABORATORY RESULT COMMENT(E141)
+CommentDate:20020330162100
+
+Interpretation: ON AZATHIOPRINE
+   </text>
+   <statusCode code="COMPLETE"/>
+   <availabilityTime value="20020330162100"/>
+  </NarrativeStatement>
+ </component>
+ <component contextConductionInd="true" typeCode="COMP">
+  <CompoundStatement classCode="CLUSTER" moodCode="EVN">
+
+   ...
+
+  </CompoundStatement>
+ </component>
+</CompoundStatement>
+```
+</details>
+
+4. A `component / CompoundStatement` is created for each `Specimen` referenced in the array 
+
+### Diagnostic Report Level Narrative Statements
+
+Narrative Statements are created to preserve data from FHIR fields that have no direct mapping.
+
+| Mapped to (XML HL7 NarrativeStatement element) | Mapped from (JSON FHIR / other source ) |
+|------------------------------------------------|-----------------------------------------|
+| id [@root]                                     | Unique ID generated by the adaptor      |
+| text [@mediaType]                              | fixed value = "text/x-h7uk-pmip"        |
+| text                                           | EDIFACT comment, as described below     |
+| statusCode                                     | fixed value = `"COMPLETE"`              |
+| availabilityTime [@value]                      | `DiagnosticReport.issued`               |
+
+#### Text Element Mapping
+
+ Narrative Statements are created in relation to each of the FHIR Fields / Logical Reasons below. The value of the field 
+ is mapped to the Narrative Statement's `text` element as EDIFACT comment (see example in the XML above), the comment type 
+ of which is shown in the table:
+
+| FHIR field / logical reason                                                                                                                                                                | EDIFACT Comment Type                                    |
```

Comment:
Each of these comments also gets a prefix, e.g. `Interpretation: `, is it worth mentioning these prefixes?

Reply:
I've added a new column